packages/create-gasket-app/lib/scaffold/actions/load-preset.js

```diff
@@ -35,7 +35,8 @@ async function loadPresets({ context }) {
       // Import the preset via the package file attrs, name and main
       // We can't specify the cwd for the import, so we need to use the full path
       // expects type:module & "main": "lib/fullpath.js"
-      const mod = await import(`${modPath}/${name}/${pkgFile.main}`);
+      const entryPath = pkgFile.main || pkgFile.exports['.'].default;
```

Comment:
These should work with ESM. Check for the import condition first, then fallback to default.
```suggestion
      const entryPath = pkgFile.main || pkgFile.exports['.'].import; || pkgFile.exports['.'].default;
```

Hindsight now, but I don't think transpiling the presets to CJS was necessary.